boards/ferris_sweep/main.py

```diff
@@ -0,0 +1,117 @@
+# Replicates default Ferris keymap from QMK
+# Credit: Pierre Chevalier, 2020
+# https://github.com/qmk/qmk_firmware/tree/master/keyboards/ferris/keymaps/default
+
+import board
+from kb import KMKKeyboard
+
+from kmk.keys import KC
+from kmk.modules.layers import Layers
+from kmk.modules.modtap import ModTap
+from kmk.modules.mouse_keys import MouseKeys
+from kmk.modules.split import Split, SplitSide
+
+keyboard = KMKKeyboard()
+
+# Adding extensions
+# rgb = RGB(pixel_pin=keyboard.rgb_pixel_pin, num_pixels=27, val_limit=100, hue_default=190, sat_default=100, val_default=5)
+
+# TODO Comment one of these on each side
+# split_side = SplitSide.LEFT
+split_side = SplitSide.RIGHT
+split = Split(
+    # split_flip=True,
+    data_pin=board.D1,
+    split_side=split_side,
+    split_target_left=False,
+    # Using the default wasn't working, try pio
+    use_pio=True,
+    uart_flip=True,
+)
+
+layers_ext = Layers()
+mod_tap = ModTap()
+mouse_key = MouseKeys()
+
+
+keyboard.modules = [layers_ext, split, mod_tap, mouse_key]
+
+# Cleaner key names
+_______ = KC.TRNS
+XXXXXXX = KC.NO
+
+
+# Mod-taps
+A_SFT = KC.MT(KC.A, KC.LSFT)
+SCLN_SFT = KC.MT(KC.SCLN, KC.LSFT)
+X_CTL = KC.MT(KC.X, KC.LCTRL)
+C_ALT = KC.MT(KC.C, KC.LALT)
+COM_ALT = KC.MT(KC.COMM, KC.LALT)
+DOT_CTL = KC.MT(KC.DOT, KC.LCTRL)
+CTL_ALT = KC.LCTRL(KC.LALT)
+
+
+# Layer tap for other home row keys
+S_L5 = KC.LT(5, KC.S)
+D_L1 = KC.LT(1, KC.D)
+F_L3 = KC.LT(3, KC.F)
+J_L4 = KC.LT(4, KC.J)
+K_L2 = KC.LT(2, KC.K)
+L_L6 = KC.LT(6, KC.L)
+SPC_L7 = KC.LT(7, KC.SPC)
+
+# fmt: off
+
+keyboard.keymap = [
+    [  # QWERTY
+    KC.Q,KC.W,  KC.E, KC.R,KC.T,  KC.Y, KC.U,KC.I,KC.O,KC.P,
+   A_SFT,S_L5,  D_L1, F_L3,KC.G,  KC.H, J_L4,K_L2,L_L6,SCLN_SFT,
+     KC.Z, X_CTL, C_ALT, KC.V,    KC.B,   KC.N, KC.M, COM_ALT, DOT_CTL,  KC.SLSH,
+        KC.LGUI,        KC.BSPC,SPC_L7,        KC.ENT,
+    ],
```

Comment:
Some of your whitespace seems to got mangled here.

Reply:
It did, thanks, should be fixed now